### PR TITLE
NHibernate timeout index doco

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
@mauroservienti the PR contains the missing piece of doco for the customer you had a case with

File: persistence/nhibernate/scripting.md [hunk 1]
```diff
@@ -26,3 +26,29 @@ Where:
 
  * `<subscriberAddress>` is the address of the subscriber. E.g. `My.Endpoint@subscriber-machine`.
  * `<subscriptionTable>` is the configured subscription table for NHibernate Persistence. By default this is `dbo.Subscription`
+
+## Fix up TimeoutEntity_EndpointIdx index
+
+Previous versions of NServiceBus had an issue which caused the `TimeoutEntity_EndpointIdx` index to be either missing or created incorrectly. The new versions have built-in check and log warnings in cases when
+ * the `TimeoutEntity_EndpointIdx` index has incorrect column order
```
Comment (HindyLambrozo) [inline persistence/nhibernate/scripting.md]: has **an** incorrect column order

File: persistence/nhibernate/scripting.md [hunk 2]
```diff
@@ -26,3 +26,29 @@ Where:
...
+ * the system could not find `TimeoutEntity_EndpointIdx` index
+
+In order to mitigate the problem use following scripts to drop the index (if created incorrectly)
```
Comment (HindyLambrozo) [inline persistence/nhibernate/scripting.md]: i suggest adding a colon at the end, including "the" and changing the order of the sentence a bit... : 

>In cases where the index was created incorrectly, use **the** following scripts to drop the index:

File: persistence/nhibernate/scripting.md [hunk 3]
```diff
@@ -26,3 +26,29 @@ Where:
...
+
+```
+DROP INDEX [TimeoutEntity_EndpointIdx] ON [dbo].[TimeoutEntity] WITH ( ONLINE = OFF )
+GO
+```
+
+and create the correct one
```
Comment (HindyLambrozo) [inline persistence/nhibernate/scripting.md]: +and **this script to** create the correct one

**Comment by SzymonPobiega** [comment]
Thanks @HindyLambrozo ❤️

**Pull request merged**